### ISSUE TITLE
[V3] Implement new Concept Exercise: constructors

### ISSUE THREAD

**Issue by kristinaborn** [exercism/javascript#953]
This issue describes how to implement the `constructors` concept exercise for the JavaScript track.

## Getting started

**Please please please read the docs before starting.** Posting PRs without reading these docs will be a lot more frustrating for you during the review cycle, and exhaust Exercism's maintainers' time. So, before diving into the implementation, please read up on the following documents:

- [The features of v3](https://github.com/exercism/v3/blob/master/docs/features-of-v3.md).
- [Rationale for v3](https://github.com/exercism/v3/blob/master/docs/rationale-for-v3.md).
- [What are concept exercise and how they are structured?](https://github.com/exercism/v3/blob/master/docs/concept-exercises.md)

Please also watch the following video:

- [The Anatomy of a Concept Exercise](https://www.youtube.com/watch?v=gkbBqd7hPrA).

## Goal

This exercise aims to teach students how constructors are implemented in JavaScript.

## Learning objectives

- Know that constructors are regular functions
- Know how to use built-in constructors
- Know how to define and use custom constructors
- Know how to use the `constructor` class method
- Understand the importance of the `new` keyword

## Out of scope

- Performance of built-in constructors vs. object literals

## Concepts

- `constructors`

## Prerequisites

- any data type used in the exercise
- `inheritance`
- `classes`

### After

- Performance of built-in constructors vs. object literals

## Help

If you have any questions while implementing the exercise, please post the questions as comments in this issue.

**Comment by junedev** [comment]
I am closing this issue because we decided to cut concepts/concept exercises differently. See https://github.com/exercism/javascript/issues/951#issuecomment-778834953 for details. Constructors are now part of the `classes` concept (https://github.com/exercism/javascript/issues/1015).
